### PR TITLE
Compatible with Magento 2.3, Disable MultiShipping, Fix UI Error

### PR DESCRIPTION
**Pull request by seherkdc**
Edit Line 2, 26, 28 in **composer.json** before commit.
Important: Change **name** to **razorpay/magento**

### CODE REVIEW

File: view/frontend/web/template/payment/razorpay-form.html [hunk 1]
```diff
@@ -4,9 +4,9 @@
...
...8MZlf3kj4j9AVZjgVVyj5l+ToYom/9k3EKosDE8ftXmqz10yxi8Pgvwaq6pEPO7SUl/kW71yzemc8HKIu6x3iMBgbXe+wdrc1Z+xtU8oOF9ThM4er6/2FjvmpJ/PIi+/HRdLJ5vW5GS14qUGhZH3qylnZUOo5N9pFglbsKoJ+FS0vv2vXCHbvy+QCjjl0wKp3u+Sc2pi6V6j4XzCVWholEOwLk/NO2ZFNXoT/r5JPd8La9ua2kOF0/XOU9wKA52rm/n7uTeT739s4NVtXdSz7nJ6m1TcuIKG/CKz/OHZ9N4RNgrs+k07UA+2xXisN0fc/apc8XV4qBlQAusNfnwA/wtb0ZuioUqbubmH6aWpd4Ip+PMGH6/LI96fSFBmjo5M7zAYRY/WA/YiRs71679Hkirf3PQXhJMF+ggsjltQDe9JX4r+xVUSiKWLTARCmkQHWOCsGO4KKrffEaoiW6Rw4sLLte0hHNAUtLDCk1+ciJT67vUNs/OOeE8xpgTpFtB4XRwWTmIdGVyzcnu8BcrhsPMcPmh+nezI3BiPtHgJaVED3c2d607mD/UHha42HwsqeCcZphnJM1HFXxD5wSx/dQobzEqKxdEApH3V8zo94eX01HIotkMOLqZswjDmh+qqO5eTh+u7P9zu2hiPsYg8+2OEiZAfAtqZ7u7+w9K/BwCfsfGYp20JNqFpbu6N5zEmBOAfMZqVT3We8UlbClH1mI+kC4aCjO5qFg2046AzAhSSrwGZ8WmOQ5vti8qesUlcRA3DT8AC5iz1wUirovBqPufT6iZRWB4KNbVy/qPpg/NS5yRUWn0zmLmE4F8xmZbpzxblGJfgwnQnOqo/n7RRdrER41DP3QDA4x41IGXxqscttAuB/Aw6OCoccPti3FYm7JS57vBLB3qgFO39WbOgdAqepgwOS1qE6Uv3JMkj2VQ04+0/Fd2d26ZKtKQlHkogUmSkEwLnJFRSf2nKiS0KD2AFjVAjZYAmtmWLuu67t3FdeKS0SIH1//wM1arH2QYMJrFryPK4gCk5aWeDYUcZ9k4Dzdee/swDAYdQDX9TEQqnLfYMJyIqwHYz0I68nve6ksU7rrrOmdPfsa0VJTszCwwesrS2W6x8FzjgTzkQAfCeYpRHSC6ct+WOU8WL9LrxXKB8aKqrnjtmzu+gMzLPsAxD4wfLob8yz0UKhlWB+A6R5YXW
```
Comment (captn3m0) [inline view/frontend/web/template/payment/razorpay-form.html]: Can we switch this to https://cdn.razorpay.com/logo.png instead?

File: Controller/Payment/Order.php [hunk 2]
```diff
@@ -36,14 +36,23 @@ public function __construct(
...
     }
 
     public function execute()
     {
         $amount = (int) (round($this->getQuote()->getBaseGrandTotal(), 2) * 100);
-
-        $receipt_id = $this->getQuote()->getId();
-
+    
+	$receipt_id = $this->getQuote()->getId();
```
Comment (captn3m0) [inline Controller/Payment/Order.php]: nit: fix alignment everywhere please

File: etc/payment.xml [hunk 3]
```diff
@@ -3,7 +3,7 @@
          xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Payment:etc/payment.xsd">
     <methods>
         <method name="razorpay">
-            <allow_multiple_address>1</allow_multiple_address>
+            <allow_multiple_address>0</allow_multiple_address>
```
Comment (captn3m0) [inline etc/payment.xml:6]: @ccrims0n Will this be a breaking change?

@seherkdc What is the impact of this?

**Review by captn3m0 — CHANGES_REQUESTED** [review]

File: etc/payment.xml (hunk 3 above)
Comment (seherkdc) [reply etc/payment.xml:6]: This disables Razorpay payment option for Multi-shipping. Magento `MultishippingController` does not support redirect URL for payment gateways. Therefore, multi-shipping needs to be disabled in the existing code.

File: view/frontend/web/template/payment/razorpay-form.html (hunk 1 above)
Comment (seherkdc) [reply view/frontend/web/template/payment/razorpay-form.html]: Yes. Use commit no. [071d048](https://github.com/kdclabs/razorpay-magento/commit/071d048788b6896b3618aeea935514b8d4e828e0#diff-38c849ad9f5324a915594a45ec6ba7c4)

**Comment by ccrims0n** [comment]
@seherkdc Can you please resolve the conflicts

**Comment by seherkdc** [comment]
@ccrims0n This is an old PR. Individual PRs for each issue have already been shared and resolved.

**Pull request closed without merging**